### ISSUE TITLE
[4.0.0 Beta 2] Debug messege flogging kodi.log

### ISSUE THREAD

**Issue by Serg86** [recalbox/recalbox-os#872]
This gets repeated many times a second in kodi.log:

`17:24:50 T:1548542960    INFO: CMMALVideo::GetPicture dts:5052.848 pts:0.000 flags:4:10000004 MMALBuffer:0x5934dc60 mmal_buffer:0x593ad6c8
17:24:50 T:1929756672   DEBUG: CMMALRenderer::FlipPage - 0
17:24:50 T:1929756672   DEBUG: CMMALRenderer::ReleaseBuffer - 1 (0x5939fa70)
17:24:50 T:1929756672   DEBUG: CMMALVideoBuffer::Release 0x5939fa70 (0x593ad140) ref:0
17:24:50 T:1929756672   DEBUG: CMMALVideoBuffer::~CMMALVideoBuffer 0x5939fa70
17:24:50 T:1548542960   DEBUG: CMMALRenderer::GetImage - 0x5c4cd9bc 1 0
17:24:50 T:1929756672   DEBUG: CMMALRenderer::RenderUpdate - 0 0 255 0
17:24:50 T:1548542960   DEBUG: CMMALRenderer::AddProcessor - 0x5934dc60 (0x593ad6c8) 1
17:24:50 T:1548542960   DEBUG: CMMALVideoBuffer::Acquire 0x5934dc60 (0x593ad6c8) ref:2
17:24:50 T:1548542960   DEBUG: CMMALRenderer::ReleaseImage - 1 0 (0x5934dc60)
17:24:50 T:1929756672   DEBUG: CMMALRenderer::RenderUpdate 0x593ad020 (0x593ad4f0) f:10000004
17:24:50 T:1929756672   DEBUG: CMMALVideoBuffer::Acquire 0x593ad020 (0x593ad4f0) ref:2
17:24:50 T:1548542960   DEBUG: CMMALVideo::SetDropState - bDrop(0)
17:24:50 T:1548542960   DEBUG: CMMALVideo::Prime - queue(0x5913e110)
17:24:50 T:1548542960   DEBUG: CMMALVideo::Recycle 0x593ad140
17:24:50 T:1548542960   DEBUG: CMMALVideo::Recycle Send buffer 0x593ad140 from pool to decoder output port 0x58fff270 ready_queue(2)
17:24:50 T:1596720112   DEBUG: CMMALRenderer::vout_input_port_cb port:0x58ffe6f0 buffer 0x593ad4f0 (0x593ad020), len 0 cmd:0 f:70000004
17:24:50 T:1596720112   DEBUG: CMMALVideoBuffer::Release 0x593ad020 (0x593ad4f0) ref:1
17:24:50 T:1548542960   DEBUG: CMMALVideo::Decode - 0x593f1408 520   /0      dts:5053.215 pts:5053.315 flags:10000004 ready_queue(2) demux_queue(0) space(1556480)
17:24:50 T:1548542960   DEBUG: CMMALVideo::Decode - got space for output: demux_queue(0) space(1556480) queued(0.47)
17:24:50 T:1548542960   DEBUG: CMMALVideo::Decode -  got output picture:2
17:24:50 T:1548542960   DEBUG: CMMALVideo::ClearPicture - 0x5934dc60 (0x593ad6c8)
17:24:50 T:1548542960   DEBUG: CMMALVideoBuffer::Release 0x5934dc60 (0x593ad6c8) ref:1
17:24:50 T:1548542960   DEBUG: CMMALVideoBuffer::Acquire 0x58f08738 (0x593ae000) ref:1`

It is flooding the file and makes it grow very large. Kodi has been running for ~4 hours and kodi.log is already over 400 MB in size. I use my Raspberry Pi 2 as an always on media center/server, therefore I spend most of the time in Kodi.

**Comment by nadenislamarre** [comment]
Thats true. It is in my todo list from a long time. But the day i disabled
it, kodi became unstable, i was not sure it was directly related. I should
redo some tests for 4.1
Le 6 avr. 2016 20:06, "Serg86" <notifications@github.com> a écrit :

This gets repeated many times a second in kodi.log:

17:24:50 T:1548542960 INFO: CMMALVideo::GetPicture dts:5052.848 pts:0.000
flags:4:10000004 MMALBuffer:0x5934dc60 mmal_buffer:0x593ad6c8
17:24:50 T:1929756672 DEBUG: CMMALRenderer::FlipPage - 0
17:24:50 T:1929756672 DEBUG: CMMALRenderer::ReleaseBuffer - 1 (0x5939fa70)
17:24:50 T:1929756672 DEBUG: CMMALVideoBuffer::Release 0x5939fa70
(0x593ad140) ref:0
17:24:50 T:1929756672 DEBUG: CMMALVideoBuffer::~CMMALVideoBuffer 0x5939fa70
17:24:50 T:1548542960 DEBUG: CMMALRenderer::GetImage - 0x5c4cd9bc 1 0
17:24:50 T:1929756672 DEBUG: CMMALRenderer::RenderUpdate - 0 0 255 0
17:24:50 T:1548542960 DEBUG: CMMALRenderer::AddProcessor - 0x5934dc60
(0x593ad6c8) 1
17:24:50 T:1548542960 DEBUG: CMMALVideoBuffer::Acquire 0x5934dc60
(0x593ad6c8) ref:2
17:24:50 T:1548542960 DEBUG: CMMALRenderer::ReleaseImage - 1 0 (0x5934dc60)
17:24:50 T:1929756672 DEBUG: CMMALRenderer::RenderUpdate 0x593ad020
(0x593ad4f0) f:10000004
17:24:50 T:1929756672 DEBUG: CMMALVideoBuffer::Acquire 0x593ad020
(0x593ad4f0) ref:2
17:24:50 T:1548542960 DEBUG: CMMALVideo::SetDropState - bDrop(0)
17:24:50 T:1548542960 DEBUG: CMMALVideo::Prime - queue(0x5913e110)
17:24:50 T:1548542960 DEBUG: CMMALVideo::Recycle 0x593ad140
17:24:50 T:1548542960 DEBUG: CMMALVideo::Recycle Send buffer 0x593ad140
from pool to decoder output port 0x58fff270 ready_queue(2)
17:24:50 T:1596720112 DEBUG: CMMALRenderer::vout_input_port_cb
port:0x58ffe6f0 buffer 0x593ad4f0 (0x593ad020), len 0 cmd:0 f:70000004
17:24:50 T:1596720112 DEBUG: CMMALVideoBuffer::Release 0x593ad020
(0x593ad4f0) ref:1
17:24:50 T:1548542960 DEBUG: CMMALVideo::Decode - 0x593f1408 520 /0
dts:5053.215 pts:5053.315 flags:10000004 ready_queue(2) demux_queue(0)
space(1556480)
17:24:50 T:1548542960 DEBUG: CMMALVideo::Decode - got space for output:
demux_queue(0) space(1556480) queued(0.47)
17:24:50 T:1548542960 DEBUG: CMMALVideo::Decode - got output picture:2
17:24:50 T:1548542960 DEBUG: CMMALVideo::ClearPicture - 0x5934dc60
(0x593ad6c8)
17:24:50 T:1548542960 DEBUG: CMMALVideoBuffer::Release 0x5934dc60
(0x593ad6c8) ref:1
17:24:50 T:1548542960 DEBUG: CMMALVideoBuffer::Acquire 0x58f08738
(0x593ae000) ref:1

It is flooding the file and makes it grow very large. Kodi has been running
for ~4 hours and kodi.log is already over 400 MB in size. I use my
Raspberry Pi 2 as an always on media center/server, therefore I spend most
of the time in Kodi.

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly or view it on GitHub
<https://github.com/recalbox/recalbox-os/issues/872>

**Comment by nadenislamarre** [comment]
It has been in my todo list for a long time.

**Comment by Serg86** [comment]
Interestingly, after deleting kodi.log while Kodi is running, it doesn't create a new file. It has been an hour already and still no kodi.log.

**Comment by nadenislamarre** [comment]
I would say it is normal
Open happen once at the kodi startup
Fwrite fails once the file is deleted
Open never rehappen.
